### PR TITLE
V2

### PR DESCRIPTION
**Pull request by rileyhales**
## Summary by CodeRabbit

- **New Features**
	- Launched a new Flask application with API endpoints, HTML pages, and CORS support.
	- Introduced versioning for API with separate modules for different versions, enhancing data product handling.
	- Added logging functionality to CloudWatch Logs for monitoring API requests.
	- Provided a comprehensive list of academic publications related to the GEOGLOWS ECMWF Streamflow Service.
- **Bug Fixes**
	- Fixed the navbar and meta descriptions for consistent branding across all templates.
- **Documentation**
	- Updated project and API documentation to reflect the standardized spelling of "GEOGLOWS."
- **Chores**
	- Improved project setup by updating `.gitignore`, Dockerfile, and supervisord configuration for better development and deployment practices.

### CODE REVIEW

File: app/v2/data.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+
+def find_available_dates() -> list:
+    forecast_zarrs = glob(os.path.join(PATH_TO_FORECASTS, "Qout*.zarr"))
+    forecast_zarrs = natsort.natsorted(forecast_zarrs, reverse=True)
+    dates = [os.path.basename(d).replace('.zarr', '').split('_')[1] for d in forecast_zarrs]
+    return dates
+
+
+def get_forecast_warnings_dataframe(date) -> pd.DataFrame:
```
Comment (msouff) [inline app/v2/data.py:60]: This should probably raise a not implemented error for now

**Review by msouff — DISMISSED** [review]
I'm planning to run this locally and take care of some of these soon, but you are welcome to address them if you can get to it before me.

File: app/v2/blueprint.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                                    hydroviewer, )
+from .controllers_historical import (retrospective,
+                                     daily_averages,
+                                     monthly_averages,
+                                     yearly_averages,
+                                     return_periods)
+from .controllers_misc import get_reach_id
+from .data import latlon_to_reach
+
+logger = logging.getLogger("DEBUG")
```
[the remark this answers is not in the document]
Comment (rileyhales) [reply app/v2/blueprint.py:23]: Debug is appropriate for this case.

File: app/v2/data.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    # find the warnings combined file using the date
+
+    # check that it exists and raise an error if not
+
+    # read and return the dataframe with pandas
+    return
+
+
+def latlon_to_reach(lat: float, lon: float) -> list:
```
[the remark this answers is not in the document]
Comment (rileyhales) [reply app/v2/data.py:73]: There is a separate geographic metadata file with lat, lon, country name, some other stuff like that.

File: app/v2/data.py (hunk 1 above)
Comment (rileyhales) [reply app/v2/data.py:60]: Do we even copy these to the web server from the compute environment?

**Comment by rileyhales** [comment]
Replaced by #22

**Pull request closed without merging**